CONTRIBUTING.md

```diff
@@ -48,7 +48,7 @@ pytest tests/test_<file_name>.py::test_<method_name>
 #### Run Flake8
 
 ```bash
-flake8 freqtrade
+flake8 freqtrade tests
```

Comment:
hmm, I guess I checked how flake is called in CI, but it was long time ago, maybe I'm wrong with it... But I think it was called as such some (~10 months) time ago...

calling simple `flake8` checks `.env`, at least at the local environment and thus takes loooong time. Maybe it also checks .env at the CI environment as well. If so, this may be a reason why it takes long time during CI runs... Calling it `flake8` at the local environment before committing leaves errors that then found by CI runs after pushing commits...

Reply:
Ci does not use virtual environment iirc - so it's not a problem for CI (flake does not take a long time there).

you're right - we should add these folders to exclude too - so lazy people (like myself) can run flake8 - without folder ... 😆

Reply:
it won't work if I (as a developer) create multiple venv, for diff. versions of python, for example.
so calling `flake8 freqtrade tests` with a concrete list of dirs is better for me. Probably same for CI, to have it checking same way...

Reply:
if you want the full list that's fine for me - but then please use the full list.
Your approach excludes `scripts` and `bin`. Both contain only one file - but if that change happens to be in that one file (especially relevant for scripts) - then this does become problematic.

That said - it reminds me to remove the binary in `bin` ... so leave that out ... as i'll remove that soon

`flake8 freqtrade tests scripts`

Reply:
okay, `flake8 freqtrade tests scripts` is okay

this is just a tip for new devs. Assume he adjusted the tests, added new tests as we wish, or simply changed types in the signatures -- he will be able to find an issue only when incorrect code pushed into github... The questions will follow, because it's the method desctribed here for devs. In the same time simple flake will check venv(s) and can even fail there... It takes very long time and questions will follow again.